### PR TITLE
Lower memory consumption.

### PR DESCRIPTION
**Pull request by alloy**
This is specifically a problem when indexing all of the user’s favourites. See the below examples:

### Before

screen shot 2015-10-09 at 22 04 51

### After

screen shot 2015-10-09 at 22 05 09

### CODE REVIEW

**Comment by alloy** [comment]
Looks good, @alloy, well done :clap:

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5026,7 +5026,6 @@
 				CLANG_WARN_OBJC_REPEATED_USE_OF_WEAK = NO;
 				CLANG_WARN_SUSPICIOUS_IMPLICIT_CONVERSION = NO;
 				CODE_SIGN_ENTITLEMENTS = Artsy/Artsy.entitlements;
-				DEBUG_INFORMATION_FORMAT = dwarf;
```
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: hrm? Wont this increase compilation times?

https://labs.spotify.com/2013/11/04/shaving-off-time-from-the-ios-edit-build-test-cycle/

File: Artsy_Tests/App_Tests/ARUserActivityAndSpotlightTests.m [hunk 2]
```diff
@@ -83,7 +83,7 @@ + (void)removeEntityByIdentifierFromSpotlightIndex:(NSString *)identifier;
             expect(attributeSet.title).to.equal(@"Artwork Title");
         });
 
-        it(@"includes a thumbnail", ^{
+        xit(@"includes a thumbnail", ^{
             expect(attributeSet.thumbnailData).to.equal(UIImagePNGRepresentation([UIImage imageNamed:@"AttentionIcon"]));
```
Comment (orta) [inline Artsy_Tests/App_Tests/ARUserActivityAndSpotlightTests.m]: is this like focus it?

File: Artsy.xcodeproj/project.pbxproj (hunk 1 above)
Comment (alloy) [inline Artsy.xcodeproj/project.pbxproj]: Surest takes a bit longer, but it has very little impact on the whole.

File: Artsy_Tests/App_Tests/ARUserActivityAndSpotlightTests.m (hunk 2 above)
Comment (alloy) [inline Artsy_Tests/App_Tests/ARUserActivityAndSpotlightTests.m]: It’s a disabled test.

File: Artsy.xcodeproj/project.pbxproj (hunk 1 above)
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: it's like a 15% decrease in overall time, it's pretty serious

Comment (alloy) [inline Artsy.xcodeproj/project.pbxproj]: Really? I’ll take another look with a clean build slate.

Comment (alloy) [inline Artsy.xcodeproj/project.pbxproj]:
```
~/C/A/eigen [master] » rm -rf DerivedData
~/C/A/eigen [master] » time xcodebuild -workspace Artsy.xcworkspace -scheme Artsy -configuration 'Debug' -sdk iphonesimulator build
       90.38 real       121.30 user        55.24 sys
~/C/A/eigen [master] » time dsymutil --out Test.dSYM DerivedData/Artsy/Build/Products/Debug-iphonesimulator/Artsy.app/Artsy
        1.61 real         1.22 user         0.27 sys
```

Where did you get the 15% number from, that article or your machine?

----

In any case, a second is a second, so if you really want I can remove it, it’s just that the next person who wants to use e.g. Instruments will have to know to set this setting. So let me know if you do prefer that?

Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: https://tech.zalando.com/blog/speeding-up-xcode-builds/

> The results were surprising: Turning on “dwarf” setting improved our Edit-Build time by 70-80%, and reduced our clean build time by 15% (22 seconds). Not bad.

Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: IMO, I think that's worth the tradeoff if it's a second on incremental builds? That's what I _really_ care about

Comment (alloy) [inline Artsy.xcodeproj/project.pbxproj]: That’s on their Pods, though. I’m guessing they have lots of them, so building the dSYM grows with it.

But sure, I’m fine with disabling it :+1:

**Pull request merged**